document/core/syntax/instructions.rst

```diff
@@ -134,15 +237,25 @@ For each type, several subcategories can be distinguished:
 
 * *Tests*: consume one operand of the respective type and produce a Boolean integer result.
 
-* *Comparisons*: consume two operands of the respective type and produce a Boolean integer result.
+* *Comparisons*: consume two operands of the respective type and produce a Boolean integer result or a result of the respective type.
 
 * *Conversions*: consume a value of one type and produce a result of another
   (the source type of the conversion is the one after the ":math:`\K{\_}`").
 
+.. todo::
+  Do these subcategories have to cover every instruction? E.g. simd shifts don't fit anywhere here, since they take 128-bit int and a 32-bit int.
```

Comment:
I think it might be clearer to group the SIMD operations separately.

Reply:
Yeah, I think that would be OK. But I'd prefer separating out comparisons too, since those work pretty differently with SIMD.

Reply:
As noted in my comment above, I think we should not lump SIMD under numeric instructions. They are different beasts. And then we can define different subcategories for them.

Reply:
Moved SIMD instructions into their own section. Called out "shift" instructions in the subcategory.
I did not add a "convention" section to add `vbinop` as a shorthand for `vibinop` and `vfbinop` yet, I think we can add those in as needed when we get to the validation chapter.